docs/architecture/tablets.rst

```diff
@@ -0,0 +1,92 @@
+=========================================
+Data Distribution  with Tablets
+=========================================
+
+A ScyllaDB cluster is a group of interconnected nodes - on-premises servers or 
+virtual servers (public cloud instances). The data of the entire cluster has to
+be distributed as evenly as possible across those nodes.
+
+ScyllaDB is designed to ensure a balanced distribution of data when adding new
+nodes (scaling out) by storing data in tablets. When you scale out, tablets are
+moved between the nodes to keep the same number on each node.
+
+This article explains the concept of tablets and how they let you scale your
+cluster quickly and seamlessly.
+
+Data Distribution
+-------------------
+
+ScyllaDB distributes data by splitting tables into tablets. Each tablet has 
+its replicas on different nodes, depending on the RF (replication factor). Each
+row of a table is mapped to a single tablet in a deterministic way. When you
+query or update the data, ScyllaDB can quickly identify the tablet that stores
+the relevant row. 
+
+The following example shows a 3-node cluster with a replication factor (RF) of
+3. The data is stored in a table (Table 1) with two rows. Both rows are mapped
+to one tablet (T1) with replicas on all three nodes.
+
+.. TODO - add the image
+
+When a tablet reaches a threshold size (measured in the number of rows),
```

Comment:
This is true in 6.0, but in the future, it will change; tablets will be split based on more triggers (like throughput)